site_scons/prereq_tools/base.py

```diff
@@ -901,6 +901,12 @@ def get_config_file(self):
     def add_options(self):
         """Add common options to environment"""
 
+        AddOption('--deps-only',
+                  dest='deps_only',
+                  action='store_true',
+                  default=False,
+                  help='Download and build dependencies only, do not build daos')
+
```

Comment:
I didn't add this, I just moved it from the SConstruct to the SConscript.   It could be improved for sure but it's used in many locations so I'd rather not change it here.

Reply:
Actually, let me upload a new patch. Looks like only a few places.

Reply:
I updated it.  The downside is dfusedaosbuild will be more broken than usual until this lands and everyone rebases.

Reply:
Yeah, it's just something I realised recently that I probably got wrong when I added it.  I guess the issue with the test is that PRs will use the code from the PR but build master so once this lands PRs that haven't been updated would fail?

The test isn't run on PRs anyway, and as it's broken it's unlikely anyone other than me would want to use it.